public/app/features/variables/adhoc/AdHocVariableEditor.test.tsx

```diff
@@ -29,17 +60,17 @@ describe('AdHocVariableEditor', () => {
   it('has a datasource select menu', async () => {
     render(<AdHocVariableEditor {...props} />);
 
-    const selectContainer = getSelectParent(screen.getByLabelText('Data source'));
-    expect(selectContainer).toHaveTextContent('Prometheus');
+    expect(await screen.findByLabelText(selectors.components.DataSourcePicker.inputV2)).toBeInTheDocument();
   });
 
   it('calls the callback when changing the datasource', async () => {
     render(<AdHocVariableEditor {...props} />);
-    await selectOptionInTest(screen.getByLabelText('Data source'), 'Loki');
+    const selectEl = screen.getByLabelText(selectors.components.DataSourcePicker.inputV2);
+    await selectOptionInTest(selectEl, 'Loki');
 
     expect(props.changeVariableDatasource).toBeCalledWith(
       { type: 'adhoc', id: 'adhoc', rootStateKey: 'key' },
-      { type: 'loki-ds', uid: 'abc' }
+      { type: 'loki', uid: 'mock-ds-2' }
     );
   });
```

Comment:
@ashharrison90  I would use some help with this failing test, for some reason the option can not be selected. What's funny is that when I change Loki to Prometheus, it selects the option as expected 🙈